mi/dataset/driver/ctdav_n/auv/ctdav_n_auv_telemetered_driver.py

```diff
@@ -9,11 +9,14 @@
 Release notes:
 
 Initial Release
+
+THIS FILE IS DEPRECATED, REPLACED BY mi/dataset/driver/ctdav_n/auv/ctdav_n_auv_driver.py
```

Comment:
This should be in the `Release notes`, not the `Initial Release` section.

Reply:
Already noted in the release notes.  Removing here.
Done.